core/templates/dev/head/pages/exploration_editor/editor_tab/ResponsesService.js

```diff
@@ -178,23 +190,24 @@ oppia.factory('ResponsesService', [
           _answerGroups = AnswerGroupsCacheService.get(newInteractionId);
         } else {
           _answerGroups = [];
-          stateContentIdsToAudioTranslationsService.displayed
-            .deleteAllFeedbackContentId();
-          stateContentIdsToAudioTranslationsService.saveDisplayedValue();
         }
-
+        updateAnswerGroupsAudioTranslation();
```

Comment:
`updateAnswerGroupAudioTranslations` might be better

Reply:
I would also suggest adding a comment briefly explaining why this is necessary. At first glance, it's not entirely obvious.

Reply:
Done!